src/components/tooltips/InformationTooltip/index.tsx

```diff
@@ -0,0 +1,32 @@
+
+import clsx from 'clsx';
```

Comment:
This is a useful bit of refactoring, but it would have been better to handle it as a separate task. Changing it in this PR involves too many changes to unrelated files and increases the amount of testing we need to do for the change (i.e. the issue and redeem forms, and the vaults table need to be retested as well as the staking UI).

I'm not asking you to back these changes out so nothing to do here, but it would be better to make sure we're not making unrelated code changes when we don't need to, especially this close to a production release. I would have preferred it if you had created the tooltip component and used it with the staking UI, but then handled the updates to the other components as a separate PR.

Reply:
> Changing it in this PR involves too many changes to unrelated files and increases the amount of testing we need to do for the change (i.e. the issue and redeem forms, and the vaults table need to be retested as well as the staking UI).

Yes, I agree with you. It sounds reasonable.
I found that the tooltip component was repeated in this staking UI and looked at other places to check it happens in other places so made that change.
I tested all updated places again and made sure that there were no bugs with this change.

I will follow your suggestion from next PRs.